FreestarAds-Ogury/Source/objc/OguryRewardMediator.m

```diff
@@ -27,17 +27,16 @@ - (BOOL)canShowRewardAd {
     return YES;
 }
 
-- (NSString*)placementId {
-    return self.mPartner.placement_id;
-}
-
 - (void)loadRewardAd {
     FSTRLog(@"OGURY: loadRewardAd");
     [self resetRewardAd];
 
-    self.ad = [[OguryOptinVideoAd alloc] initWithAdUnitId:[self placementId]];
-    [self.ad load];
+    FSTRLog(@"OGURY: placement_id %@", self.mPartner.placement_id);
+    FSTRLog(@"OGURY: adunitId %@", [self.mPartner adunitId]);
+
+    self.ad = [[OguryOptinVideoAd alloc] initWithAdUnitId:[self.mPartner adunitId]];
```

Comment:
Same here, should check adunit id for nil and if so, call partnerAdLoadFailed:

Reply:
Fixed